### DISCUSSION TITLE
Formatting the #_RSVPEND placeholder?

### DISCUSSION THREAD

**Discussion by Schneleed** [liedekef/events-made-easy#476] [Q&A] [answered]
Hi everyone,

is it possible to use the same formatting logic for the #_RSVPEND placeholder than we can use for other date-placeholders like #_STARTDATE{d.m.Y}

Thanks for any hint on this topic in advance!

**Comment by liedekef** [comment]
Currently it is not possible, feel free to open a feature request for this

**Reply by liedekef** [reply, marked as answer]
Btw, from the changelog of the next version:
* Added support for event placeholders #_RSVPSTART{xx} and #_RSVPEND{xx}, next to just #_RSVPSTART and #_RSVPEND so you can format the layout to your liking ('xx' being the date/time format in php date notation)

**Reply by Schneleed** [reply]
So no need for another feature request as it is already included in the next release, correct?
